NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java

```diff
@@ -322,4 +322,46 @@ private static void disableFallAnimation() {
         c.addComment("settings.disabled-block-fall-animation", "Disables the fall animation for blocks");
     }
 
+    public static boolean disableInfinitSleeperThreadUsage;
+
+    private static void disableInfinitSleeperThreadUsage() {
+        disableInfinitSleeperThreadUsage = getBoolean("settings.disable-infinisleeper-thread-usage", false);
+        c.addComment("settings.disable-infinisleeper-thread-usage", "Disable infinisleeper thread usage, just enable this if you know what are you doing.");
+    }
+
+    public static boolean enableFastMath;
+
+    private static void enableFastMath() {
+        enableFastMath = getBoolean("settings.enable-fastmath", false);
+        c.addComment("settings.enable-fastmath", "Enable Fast Math usage.");
+    }
+
+    public static boolean enableFastMathCosSin;
+
+    private static void enableFastMathCosSin() {
+        enableFastMathCosSin = getBoolean("settings.enable-fastmath-cos-sin", false);
+        c.addComment("settings.enable-fastmath-cos-sin", "Enable Fast Math usage with cos() and sin() methods, this may break anticheats and server-side calculations.");
+    }
+
+    public static int titleEntityTickingTime;
+
+    private static void titleEntityTickingTime() {
+        titleEntityTickingTime = getInt("settings.tile-entity-ticking-time", 20);
+        c.addComment("settings.tile-entity-ticking-time", "Ticking time in seconds for usage on tile entity operations.");
+    }
+
+    public static int itemDirtyTicks;
+
+    private static void itemDirtyTicks() {
+        itemDirtyTicks = getInt("settings.item-dirty-ticks", 20);
+        c.addComment("settings.item-dirty-ticks", "Change the behavior of dirty itemstacks checking.");
```

Comment:
Is it possible to elaborate a bit more on what this does? (in the config comment)